images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -160,3 +160,31 @@ Hashtable for service arguments
         }
     }
 }
+
+function Get-VS19ExtensionVersion
+{
+    param (
+        [string] [Parameter(Mandatory=$true)] $packageName
+    )
+
+    $vsProgramData = Get-Item -Path "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances"
```

Comment:
What is the purpose Get-Item -Path ? We already have fullName.

Reply:
Thanks, removed